client/app/dedicated/server/reverse-dns/delete/reverse-dns-delete.html

```diff
@@ -0,0 +1,24 @@
+<div class="oui-modal oui-modal_shadow">
+    <form name="deleteReverseDNSForm"
+          novalidate
+          data-ng-submit="$ctrl.onDeleteReverseDNSFormSubmit()">
+        <oui-modal
+            data-title="{{ :: 'server_reversedns_delete_title' | translate }}"
+            data-primary-label="{{($ctrl.actionEnabled ? 'server_reversedns_delete_confirm' : 'global_OK') | translate }}"
+            data-secondary-label="{{$ctrl.actionEnabled ? ('common_cancel' | translate) : '' }}"
+            data-secondary-action="$ctrl.$state.go('^')"
+            data-on-dismiss="$ctrl.$state.go('^')"
+            data-loading="$ctrl.loading.init || $ctrl.loading.delete"
+            data-type="{{ !$ctrl.actionEnabled ? 'warning' : '' }}">
+
+            <div data-ng-if="!$ctrl.actionEnabled">
+                <p data-translate="server_reversedns_delete_loading_error"></p>
```

Comment:
You can apply `data-ng-if` directly on your `<p>` element (same for next lines).